shared/js/background/tab-manager.es6.js

```diff
@@ -118,8 +118,7 @@ class TabManager {
         if (tab) {
             tab.statusCode = request.statusCode
             if (tab.statusCode === 200) {
-                tab.url = request.url
-                tab.updateSite()
```

Comment:
Moved setting the URL within `updateSite` so I don't have to do it in multiple places.